CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+# Essential Scala Collaborators
+
+Congratulations! You have been selected as a content owner for the  **Scala Essentials** nanodegree.
+
+
+## About the nanodegree
+
+The **Scala Essentials** nanodegree aims to provide a comprehensive tour of the Scala Programming Language for begginers. 
+
+You can see the design document [here](https://docs.google.com/document/d/1HZ1Rsn46x_BVAo8scSyVr9r5IesyTyHhGptjW1n_T6A/edit?usp=sharing).
+
+**Modules**:
+1. Introduction
+2. Essentials
+3. Objects and classes
+4. Functions
+5. Traits for data modelling
+6. Sequencing computation
+7. Idiomatic Scala
+8. Type classes
+9. Functional programming
+10. Popular frameworks and toolkits
+
+## About content owners
+
+As a content owner your job is to design and create the content of at least one module. You can build a small team with other content owners to assist on the content creation. Please consider that each conent owner has full responsibility of their own modules. 
+
+Content owners:

```

Comment:
What about adding a contact method, email maybe?, of every contributor.

Reply:
We are thinking about using Gitter as the official communication channel for this Nanodegree (will add the gitter badge soon). Nonetheless, I agree on providing individual contact method for each collaborator.

Reply:
We can use the gitter private messaging url for each collaborator (e.g. https://gitter.im/oscarvarto).